src/queue.py

```diff
@@ -17,18 +17,17 @@ def enqueue(self, value):
         """Add value to the top of  the queue."""
         return self._dlist.push(value)
 
-    def dequeue(self, value):
+    def dequeue(self):
```

Comment:
We didn't need to pass in anything for the dequeue function.  It will just dequeue the object that has been in the queue the longest.